cmd/nitro/nitro.go

```diff
@@ -146,6 +148,9 @@ func mainImpl() int {
 		fmt.Fprintf(os.Stderr, "Error initializing logging: %v\n", err)
 		os.Exit(1)
 	}
+	if nodeConfig.Node.Pprof.Enable {
```

Comment:
Sorry about the pivot, but now that I've read through the existing geth code, how about something completely different:  If metrics is enabled but pprof is disabled, startup metrics server as in the existing code.  If `--node.pprof` is specified, then call  `debug.StartPProf(address, metrics_enabled)` from `"github.com/ethereum/go-ethereum/internal/debug"`, that way pprof doesn't have to define yet another port.  It isn't ideal that pprof defines handlers is `DefaultServeMux`, but it doesn't look like any of the HTTP handlers default to `DefaultServeMux`, so we should be good there.